### ISSUE TITLE
Firmware copy fails in PostStartHook

### ISSUE THREAD

**Issue by tkatila** [kubernetes-sigs/kernel-module-management#511]
I was trying the latest version yesterday and I came across an error with a previously working setup:
```
          Exec lifecycle hook ([/bin/sh -c cp -r "/intel-dgpu-firmware/*" /var/lib/firmware && modprobe -v -d "/opt" "i915"]) for Container "module-loader" in Pod "kmm-i915-backport-s87cs-7xjqz_default(9f1a7568-389a-4d0e-b413-9ffe1307fe5f)" failed - error: command '/bin/sh -c cp -r "/intel-dgpu-firmware/*" /var/lib/firmware && modprobe -v -d "/opt" "i915"' exited with 1: cp: cannot stat '/intel-dgpu-firmware/*': No such file or directory
          , message: "cp: cannot stat '/intel-dgpu-firmware/*': No such file or directory\n"
        reason: PostStartHookError
```
If I run the command manually without the quotes, it succeeds. Also, if I downgrade the operator container to 1.1.0 version, it succeeds. Maybe the star(*) should be outside of the quotes?

**Comment by qbarrand** [comment]
The `main` branch is currently unstable as we are working towards 2.0. I would recommend that you remain on a released version, such as 1.1.0.

**Comment by qbarrand** [comment]
This should be fixed in 2.0; closing.
